RiotShareExtension/ViewController/ShareViewController.m

```diff
@@ -94,7 +94,7 @@ - (void)configureViews
     
     [self resetContentView];
     
-    if ([ShareExtensionManager sharedManager].mxSession)
+    if ([MXKAccountManager sharedManager].activeAccounts.count)
```

Comment:
Check here the`account` (or `userAccount`) property of the [ShareExtensionManager sharedManager].